### ISSUE TITLE
Create a new logo and icon design for gngr

### ISSUE THREAD

**Issue by hrj** [UprootLabs/gngr#221]
The current logo and icon for gngr is crude. It was designed by myself in a hurry and I am not quite happy with it.

`gngr` needs a better designed logo. It needs to convey gngr's concept better. It would be great if it is distinctive, memorable and attractive.

`gngr` is short for "ginger", the spice. It has a connotation of "being cautious".

If a logo idea is complex, a simpler "icon" version of it would be desirable.

**Comment by hrj** [comment]
How is this for a start:

![gngr](https://cloud.githubusercontent.com/assets/345879/20839897/4f74003a-b8d4-11e6-95f1-2047d0bd9f2d.png)

**Comment by arkany** [comment]
Were you looking for something literal, similar to how Tor uses an onion?
<img width="562" alt="gngr-literal" src="https://cloud.githubusercontent.com/assets/192032/22814314/6280ae08-ef07-11e6-9cc0-7dc7886dea2f.png">

**Comment by hrj** [comment]
@arkany Yeah, literal would be good. Thanks for submitting. But I am not sure if going literal is a good idea here. Onions are nice and symmetrical; a ginger would need some aesthetic makeover :) Or, say, the ginger forming a shape relevant to "browsing".

**Comment by beardedpayton** [comment]
@hrj proportional, simple and clean. Is this on the right track for your open source project?

![screen shot 2017-02-20 at 1 39 37 pm](https://cloud.githubusercontent.com/assets/20526900/23137792/174963ee-f772-11e6-96b4-cb7612ab769b.png)

![screen shot 2017-02-20 at 1 37 34 pm](https://cloud.githubusercontent.com/assets/20526900/23137794/1d253f7c-f772-11e6-9695-0dc8d177533d.png)

**Comment by hrj** [comment]
@beardedpayton Yes, I am liking the direction there! I like the colors of the second one more.

Would be great if you, @arkany (and anyone else interested) could join our [gitter chat room](https://gitter.im/UprootLabs/gngr) or the [matrix room](https://vector.im/beta/#/room/#freenode_#gngr:matrix.org). I think it would be easier to discuss in chat form.

**Comment by beardedpayton** [comment]
@hrj sounds good. We can definitely explore colors more.

**Comment by ajmos** [comment]
![gngr](https://cloud.githubusercontent.com/assets/25403432/23323968/24cdbf34-fab9-11e6-8572-e927e79f642b.png)

**Comment by hrj** [comment]
Update: I had originally thought we can have multiple submissions on the issue list, and choose one based on votes. However, we don't seem to get a lot of visibility here to have a good number of votes. Also it is hard to predict who will contribute submissions and when.

I started discussing with @beardedpayton on chat and he was quick to iterate with me on several ideas. So, I have assigned this issue to him, and letting him drive this effort.

**Comment by beardedpayton** [comment]
Core assets have been sent to @hrj 

Below after a lot of variations is the direction we decided to go with. 
A lot of logo/brand exploration took place to figure out what exactly fit gngr best. Privacy and Security were two big points we decided to focus on. 

Biggest issue was trying to associate ginger root colors and the insides while focusing on our two big points. I think everything turned out well and will be an awesome logo for the brand/project!

![gngr](https://cloud.githubusercontent.com/assets/20526900/23416545/40a142c6-fdb1-11e6-8229-7de9761252af.png)

**Comment by hrj** [comment]
@beardedpayton Belated thanks for your work! I had time this week to work on gngr again, and planning to the use the logos into the product and website. Can you please specify the license of your contribution? I suggest [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/).

**Comment by beardedpayton** [comment]
@hrj Logo is under  (CC BY 4.0) license. You are free to use it as long as appropriate credit is given to me where applicable. Looking forward to seeing gngr up and running!

**Comment by hrj** [comment]
I was trying out something today. 👍🏼 or 👎🏼  ?

![gngrMockup](https://user-images.githubusercontent.com/345879/104050614-e3363d80-520c-11eb-8a14-3794762ab5e5.png)

**Comment by Erioldoesdesign** [comment]
Hey looks like this thread is active but is it ok to close the job issue on open source design from 2016? https://opensourcedesign.net/jobs/jobs/2016-10-22-gngr-logo

**Comment by hrj** [comment]
@Erioldoesdesign Yes, you can close. I still want to circle back on the logo ideas though. Maybe I can open a new job issue later. Thank you.

**Comment by Erioldoesdesign** [comment]
> @Erioldoesdesign Yes, you can close. I still want to circle back on the logo ideas though. Maybe I can open a new job issue later. Thank you.

Please do consider opening a new job when you need design help again 😄
